pkg/apis/datavolumecontroller/v1alpha1/types.go

```diff
@@ -42,6 +42,8 @@ type DataVolumeSpec struct {
 	Source DataVolumeSource `json:"source"`
 	//PVC is a pointer to the PVC Spec we want to use
 	PVC *corev1.PersistentVolumeClaimSpec `json:"pvc"`
+	//ContentType options: "kubevirt", "archive"
+	ContentType string `json:"contentType,omitempty"`
```

Comment:
I think it would be better to create a new type here.  So something like this:

```
type DataVolumeContentType string

const (
    DataVolumeKubeVirt = "kubevirt"
    DataVolumeArchive = "archive"
)
```

Reply:
@mhenriks so the following consts are defined in pkg/controller/util.go:
// ContentTypeKubevirt is the content-type of the import, defaults to kubevirt
ContentTypeKubevirt = "kubevirt"
// ContentTypeArchive is the content-type to specify if wanting to extract an archive
ContentTypeArchive = "archive"

and there are several references to these two, from other files. Should I remove these two consts from utils.go as well and use the new consts I'm going to add in types.go instead?

Reply:
Yeah I think using api type definitions would be best.

Reply:
Done